src/v/cluster/controller.h

```diff
@@ -120,11 +120,13 @@ class controller {
 
     /**
      * Create raft0, and start the services that the \c controller owns.
+     * \param initial_raft0_brokers Brokers to start raft0 with. Empty for
+     *      non-seeds.
      * \param stored_cluster_uuid Cluster UUID as it appears in the kvstore,
      *      or empty if it does not.
      */
     ss::future<> start(
-      std::vector<model::broker>,
+      std::vector<model::broker>&& initial_raft0_brokers,
```

Comment:
why is this change required ?, if we need it we should make it a separate commit